doc/v2.rst

```diff
@@ -94,6 +94,18 @@ Please instantiate the objects instead.
 The ``update_endpoint`` argument in ``deploy()`` methods for estimators and models has been deprecated.
 Please use :func:`sagemaker.predictor.Predictor.update_endpoint` instead.
 
+``content_type`` and ``accept`` in the Predictor Constructor
+------------------------------------------------------------
+
+The ``content_type`` and ``accept`` parameters have been removed from the
+following methods:
+- ``sagemaker.predictor.Predictor.__init__``
```

Comment:
instead of writing out `__init__` I might just say "the following methods and classes"

Reply:
Fixed